web/controllers/search.ex

```diff
@@ -0,0 +1,43 @@
+defmodule App.Search do
+  @moduledoc """
+    Functions for searching through resources with a query and checking if query
+    terms are in resource anywhere. Excludes stop words ('a', 'and', 'the' etc.)
+    and takes typos into account. 
+  """
+  @stop_words "stop_words.txt" |> File.read! |> String.split(",")
```

Comment:
I don't think it's an issue but I was wondering if using ```File.read!``` for each search might be a bit slower than loading the stop keyword in a config variable just once?